src/views/Movie/Movie.test.jsx

```diff
@@ -0,0 +1,36 @@
+import { screen, render } from '@testing-library/react';
+import Movie from './Movie';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { castle } from '../../utils/castle';
+import { MemoryRouter } from 'react-router-dom';
+
+const server = setupServer(
+  rest.get(`https://ghibliapi.herokuapp.com/films/:id`, (req, res, ctx) => {
+    return res(ctx.json(castle));
+  })
+);
+
+// 🚨 Listen for server start
+beforeAll(() => server.listen());
+
+// 🚨 Close server when complete
+afterAll(() => server.close());
+
+test('Should be able to render Castle in the Sky', async () => {
+  render(
+    <MemoryRouter initialEntries={['/2baf70d1-42bb-4437-b551-e5fed5a87abe']}>
+      <Movie />
```

Comment:
You're fine here, but be aware that you can run into trouble if you don't wrap the component in its route with path. 
` <Route exact path="/:movieId">
        <Movie />
   </Route>`